cmd/infra/aws/create.go

```diff
@@ -169,3 +195,15 @@ func AWSClient(creds, region string) (ec2iface.EC2API, error) {
 	}
 	return ec2.New(s), nil
 }
+
+func route53Client(creds, region string) (route53iface.Route53API, error) {
+	awsConfig := &aws.Config{
+		Region: aws.String(region),
```

Comment:
This may not work in surprising ways: https://github.com/openshift/cluster-ingress-operator/blob/master/pkg/dns/aws/dns.go#L146

Reply:
I think for now hard-coding to `us-east-1` will be good enough so that tag queries consistently work